tools/NuGet/template-nuget/DynamoVisualProgramming.ZeroTouchLibrary.nuspec

```diff
@@ -25,6 +28,9 @@
         <file src="ProtoGeometry.dll" target="lib\$TargetFramework$" />
         <file src="ProtoGeometry.xml" target="lib\$TargetFramework$" />
```

Comment:
and these two lines?

Reply:
they are needed for the net8 configurations
Nuget does not know to pick some assemblies form netstandard and the rest from net8. It goes for one targetframework or the other. So both need all assemblies

Reply:
So does this nuspec create a nuget package targeting net8 containing all these assemblies and also targeting netstandard2 with only protogeometry and dynamoservices assemblies?

Reply:
Yes

Reply:
It will allow some zero touch packages to target nestandard2.0 so that it can be compatible with both Dynamo 2.19 and Dynamo 3.0